pontoon/base/migrations/0100_bug_1390805_create_missing_tm_entries.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-08-16 11:37
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def create_missing_translaton_memory_entries(apps, schema_editor):
+    Translation = apps.get_model('base', 'Translation')
+    TranslationMemoryEntry = apps.get_model('base', 'TranslationMemoryEntry')
+
+    translations_to_create_translaton_memory_entries_for = (
+        Translation.objects
+        .filter(approved=True, memory_entries__isnull=True)
+        .prefetch_related('entity__resource__project')
```

Comment:
`values_list` could be a little less memory/time consuming if that is a problem.